modules/serverless-creating-a-kafka-event-sink.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-event-sinks.adoc
+
+:_content-type: PROCEDURE
+[id="serverless-creating-a-Kafka-event-sink_{context}"]
+
+= Creating a Kafka event sink
+
+As a developer, you can create an event sink to receive events from a particular source, such as a bootstrap server and send them to a kafka topic. 
```

Comment:
```suggestion
As a developer, you can create an event sink to receive events from a particular source and send them to a kafka topic. 
```

Reply:
I'd leave out the "bootstrap server" out because it's misleading. The KafkaSink is there for receiving events and sending them to a Kafka cluster that is identified by bootstrap servers and the target kafka topic. It's NOT supposed to get events "from" bootstrap servers.

The rest of this PR looks good.

Reply:
Thank you @mgencur for your suggestion. Incorporated it, please verify the same and provide an ACK.